projects/opentelemetry-interceptor/package.json

```diff
@@ -17,17 +17,17 @@
   "peerDependencies": {
     "@angular/common": ">=9.0.0",
     "@angular/core": ">=9.0.0",
-    "@opentelemetry/exporter-collector": "0.24.0",
-    "@opentelemetry/web": "0.24.0",
-    "@opentelemetry/exporter-zipkin": "0.24.0",
-    "@opentelemetry/propagator-jaeger": "0.24.0",
-    "@opentelemetry/propagator-b3": "0.24.0",
-    "@opentelemetry/api": "1.0.1",
-    "@opentelemetry/context-zone-peer-dep": "0.24.0",
-    "@opentelemetry/instrumentation": "0.24.0",
-    "@opentelemetry/instrumentation-document-load": "0.24.0",
-    "@opentelemetry/instrumentation-fetch": "0.24.0",
-    "@opentelemetry/instrumentation-xml-http-request": "0.24.0"
+    "@opentelemetry/exporter-collector": "0.25.0",
+    "@opentelemetry/sdk-trace-web": "0.25.0",
+    "@opentelemetry/exporter-zipkin": "0.25.0",
+    "@opentelemetry/propagator-jaeger": "0.25.0",
+    "@opentelemetry/propagator-b3": "0.25.0",
+    "@opentelemetry/api": "1.0.2",
+    "@opentelemetry/context-zone-peer-dep": "0.2(.0",
+    "@opentelemetry/instrumentation": "0.2(.0",
```

Comment:
hey @jufab, are these version numbers correct?

Reply:
Oops...

Thx, i'm going to fix it!